simple_history/models.py

```diff
@@ -343,13 +347,28 @@ def create_historical_record(self, instance, history_type):
         history_user = self.get_history_user(instance)
         history_change_reason = getattr(instance, 'changeReason', None)
 
+        pre_create_historical_record.send(
+            sender=self.__class__, instance=instance,
```

Comment:
The `sender` should be the `HistoricalModelClass` not `HistoricalRecords`, no? I think by saying `self.__class__` it ends up being `HistoricalRecords`.

Reply:
That makes sense. I will update the code :)